mysql_scripts/table_creation/FullDB.sql

```diff
@@ -4,6 +4,8 @@ DROP TABLE IF EXISTS Answer;
 DROP TABLE IF EXISTS AppointmentFilingStatus;
 DROP TABLE IF EXISTS FilingStatus;
 DROP TABLE IF EXISTS ServicedAppointment;
+DROP TABLE IF EXISTS AppointmentClientReschedule;
```

Comment:
Not sure if this is a good name for this table... thoughts? Maybe something like AppointmentRescheduleToken? This table holds the token and appointmentId right now. And in the future maybe failedValidationCount/lockoutTime if we deem it necessary to have security / throttling around thee self-service reschedule stuff. @CollinsSpencer @JakeKo

Reply:
I don't have a problem with the `AppointmentClientReschedule` name, but I think `AppointmentRescheduleToken` might better fit the naming pattern with nouns for table names.